Makefile

```diff
@@ -130,16 +130,13 @@ test:
 	@echo
 	go test -tags "${NOTARY_BUILDTAGS}" $(TESTOPTS) $(PKGS)
 
-test-full: TESTOPTS =
```

Comment:
Combined all the lint functionality into one function, as per https://github.com/docker/notary/pull/830#issuecomment-232515700